app/src/main/java/com/metro/gwuexplorer/MainActivity.kt

```diff
@@ -90,54 +69,74 @@ class MainActivity : AppCompatActivity() {
 //                    x++
 //                }
 
-                    val first = results[0]
-                    val firstAdd = first.getAddressLine(0)
-//
-//                //val secondAdd = first.getAddressLine(1)
-//               // Log.d("second Add:", secondAdd)
-//              // val  addr: List<String> = listOf(firstAdd)
-//
-                    addr = mutableListOf(firstAdd)
-
+                first = results[0]
```

Comment:
The intention was for the dialog to display the _address results_ (not the possible entrances).

I suppose this is okay as-is though, same concepts are being applied either way.